TicTacToe2/TicTacToeGame.cs

```diff
@@ -8,7 +8,31 @@ class TicTacToeGame
     {
         public void Play()
         {
+            bool winner = false;
+            while (!winner)
+            {
+                DrawBoard();
+                UserInput();
+                winner = CheckWinDraw();
+                ChangePlayer();
+            }
+        }
 
+        private void DrawBoard()
+        {
+            throw new NotImplementedException();
+        }
+        private void UserInput()
```

Comment:
Function names typically describe actions. I recommend changing this function name to GetUserInput